news/_posts/core-weekly/2019-06-27-coreweekly-week-25-2019.md

```diff
@@ -0,0 +1,155 @@
+* [#14248](https://github.com/PrestaShop/PrestaShop/pull/14248): Cannot use accented URL with arabic language, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#14259](https://github.com/PrestaShop/PrestaShop/pull/14259): Avoid bug when creating manually created a language, by [@jolelievre](https://github.com/jolelievre)
+* [#14290](https://github.com/PrestaShop/PrestaShop/pull/14290): Extend customer page, by [@jolelievre](https://github.com/jolelievre)
+* [#14292](https://github.com/PrestaShop/PrestaShop/pull/14292): Update php-cs-fixer to latest version. Thank you [@MathiasReker](https://github.com/MathiasReker)
+
+
+### Front office
+
+* [#14251](https://github.com/PrestaShop/PrestaShop/pull/14251): Fix categories wordbreak, by [@jolelievre](https://github.com/jolelievre)
+* [#14296](https://github.com/PrestaShop/PrestaShop/pull/14296): Fix carrier display in order confirmation email, by [@jolelievre](https://github.com/jolelievre)
+* [#14298](https://github.com/PrestaShop/PrestaShop/pull/14298): Fix english mail generation on fresh install, by [@jolelievre](https://github.com/jolelievre)
+* [#14309](https://github.com/PrestaShop/PrestaShop/pull/14309): Simplify the installation language to its minimum, by [@jolelievre](https://github.com/jolelievre)
+* [#14325](https://github.com/PrestaShop/PrestaShop/pull/14325): Refresh product flags on ajax call when product page is refreshed, by [@matks](https://github.com/matks)
+
+
+### Install
+
+* [#14225](https://github.com/PrestaShop/PrestaShop/pull/14225): Update facetedsearch cache, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#14243](https://github.com/PrestaShop/PrestaShop/pull/14243): Fix translation catalogue, by [@eternoendless](https://github.com/eternoendless)
+
+
+### Tests
+
+* [#13999](https://github.com/PrestaShop/PrestaShop/pull/13999): Fix broken tests. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#14187](https://github.com/PrestaShop/PrestaShop/pull/14187): Minor fixes for E2E tests (adding pauses, adding async code, referencing blocking issues), by [@SimonGrn](https://github.com/SimonGrn)
+* [#14219](https://github.com/PrestaShop/PrestaShop/pull/14219): Fix test Module '04_sort_module' in suite full, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#14221](https://github.com/PrestaShop/PrestaShop/pull/14221): Fix "14_discount/3_check_vouchers_in_FO" in full campaign. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#14282](https://github.com/PrestaShop/PrestaShop/pull/14282): Fix download in E2E tests, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+## Code changes in modules, themes & tools
+
+### Translation tools bundle
+
+
+
+### Auto upgrade
+
+* [#322](https://github.com/PrestaShop/autoupgrade/pull/322): Fix warning in 1.7 when settings file does not exist. Thank you [@jf-viguier](https://github.com/jf-viguier)
```

Comment:
Is it worth mentioning, as the 2 PR cancels each other ?

Reply:
the fact is that they are counted and after all it's the result of work, tests, choices. that's interesting: if it had been reverted there's a reason why. and we should not hide that.